integrations/webchat/contributing/template/client/react/src/config.js

```diff
@@ -1,9 +1,12 @@
-// A web chat configuration options object as documented at https://web-chat.global.assistant.watson.cloud.ibm.com/docs.html?to=api-configuration#configurationobject
+// This is the standard web chat configuration object. You can modify these values with the embed code for your
+// own assistant if you wish to try this example with your assistant. You can find the documentation for this at
+// https://web-chat.global.assistant.watson.cloud.ibm.com/docs.html?to=api-configuration#configurationobject.
 const config = {
-  integrationID: '',
-  region: '',
-  serviceInstanceID: '',
-  subscriptionID: '',
+  // TODO: Update this with the assistant configured for this example.
+  integrationID: 'xyz',
+  region: 'us-south',
```

Comment:
should we make region and service instance dummy values, too?

Reply:
I don't think so. Because we're (hopefully) going to  be putting all the assistants in the same instance, those values should be the same for every tutorial.